pajbot/modules/__init__.py

```diff
@@ -103,6 +102,7 @@
     LineFarmingModule,
     LinkCheckerModule,
     LinkTrackerModule,
+    ChattersRefreshModule,
```

Comment:
remove this resort - it is wrong. this list is sorted based on the name of the class, not any of its internal members